added kernel polling each second for 10 minutes

sigmarkarl opened the pull request:
added kernel polling

sigmarkarl committed:
added kernel polling each second for 10 minutes

ImpSy commented on packages/services/src/session/restapi.ts:
```diff
@@ -100,16 +100,37 @@ export async function startSession(
 ): Promise<Session.IModel> {
   const url = URLExt.join(settings.baseUrl, SESSION_SERVICE_URL);
-  const init = {
+  const body = JSON.stringify(options)
+  const bodyjson = JSON.parse(body);
```
you should online this to better show the intent and call it something like optionsCopy

ImpSy commented on packages/services/src/session/restapi.ts:
```diff
@@ -100,16 +100,37 @@ export async function startSession(
-    const err = await ServerConnection.ResponseError.create(response);
+  let data = {"id": "waiting", "session_id": ""};
+  let count = 0
+  while (count++ < 600) {
```
you never increment count

ImpSy commented on packages/services/src/session/restapi.ts:
```diff
@@ -100,16 +100,37 @@ export async function startSession(
-  const response = await ServerConnection.makeRequest(url, init, settings);
-  if (response.status !== 201) {
-    const err = await ServerConnection.ResponseError.create(response);
+  let data = {"id": "waiting", "session_id": ""};
```
why define data here ?

sigmarkarl committed:
added workflow.yml

sigmarkarl committed:
ocean jupyter lab

sigmarkarl committed:
Publish 3.6.6

SHA256 hashes:

ocean_jupyterlab-3.6.6-py3-none-any.whl: 260fc8be5e0a660829ebdfcfef86d89d1a9adab1693115b774e97a9c5642447b

ocean_jupyterlab-3.6.6.tar.gz: b070b15c358e2e4c16a8ffa001786d775b2d6fd2109f5ba098e3cdec89e11bab

sigmarkarl committed:
[ci skip] New version

 - @jupyterlab/builder@3.6.7
 - @jupyterlab/buildutils@3.6.7
 - @jupyterlab/template@3.6.7
 - @jupyterlab/application-top@3.6.7
 - @jupyterlab/example-app@3.6.7
 - @jupyterlab/example-cell@3.6.7
 - @jupyterlab/example-console@3.6.7
 - @jupyterlab/example-federated@2.7.7
 - @jupyterlab/example-filebrowser@3.6.7
 - @jupyterlab/example-notebook@3.6.7
 - @jupyterlab/example-terminal@3.6.7
 - @jupyterlab/example-federated-core@2.7.7
 - @jupyterlab/example-federated-md@2.7.7
 - @jupyterlab/example-federated-middle@2.7.7
 - @jupyterlab/example-federated-phosphor@2.7.7
 - @jupyterlab/galata@4.5.7
 - @jupyterlab/mock-extension@3.6.7
 - @jupyterlab/mock-consumer@3.6.7
 - @jupyterlab/mock-provider@3.6.7
 - @jupyterlab/mock-token@3.6.7
 - @jupyterlab/application-extension@3.6.7
 - @jupyterlab/application@3.6.7
 - @jupyterlab/apputils-extension@3.6.7
 - @jupyterlab/apputils@3.6.7
 - @jupyterlab/attachments@3.6.7
 - @jupyterlab/cell-toolbar-extension@3.6.7
 - @jupyterlab/cell-toolbar@3.6.7
 - @jupyterlab/cells@3.6.7
 - @jupyterlab/celltags-extension@3.6.7
 - @jupyterlab/celltags@3.6.7
 - @jupyterlab/codeeditor@3.6.7
 - @jupyterlab/codemirror-extension@3.6.7
 - @jupyterlab/codemirror@3.6.7
 - @jupyterlab/collaboration-extension@3.6.7
 - @jupyterlab/collaboration@3.6.7
 - @jupyterlab/completer-extension@3.6.7
 - @jupyterlab/completer@3.6.7
 - @jupyterlab/console-extension@3.6.7
 - @jupyterlab/console@3.6.7
 - @jupyterlab/coreutils@5.6.7
 - @jupyterlab/csvviewer-extension@3.6.7
 - @jupyterlab/csvviewer@3.6.7
 - @jupyterlab/debugger-extension@3.6.7
 - @jupyterlab/debugger@3.6.7
 - @jupyterlab/docmanager-extension@3.6.7
 - @jupyterlab/docmanager@3.6.7
 - @jupyterlab/docprovider-extension@3.6.7
 - @jupyterlab/docprovider@3.6.7
 - @jupyterlab/docregistry@3.6.7
 - @jupyterlab/documentsearch-extension@3.6.7
 - @jupyterlab/documentsearch@3.6.7
 - @jupyterlab/extensionmanager-extension@3.6.7
 - @jupyterlab/extensionmanager@3.6.7
 - @jupyterlab/filebrowser-extension@3.6.7
 - @jupyterlab/filebrowser@3.6.7
 - @jupyterlab/fileeditor-extension@3.6.7
 - @jupyterlab/fileeditor@3.6.7
 - @jupyterlab/help-extension@3.6.7
 - @jupyterlab/htmlviewer-extension@3.6.7
 - @jupyterlab/htmlviewer@3.6.7
 - @jupyterlab/hub-extension@3.6.7
 - @jupyterlab/imageviewer-extension@3.6.7
 - @jupyterlab/imageviewer@3.6.7
 - @jupyterlab/inspector-extension@3.6.7
 - @jupyterlab/inspector@3.6.7
 - @jupyterlab/javascript-extension@3.6.7
 - @jupyterlab/json-extension@3.6.7
 - @jupyterlab/launcher-extension@3.6.7
 - @jupyterlab/launcher@3.6.7
 - @jupyterlab/logconsole-extension@3.6.7
 - @jupyterlab/logconsole@3.6.7
 - @jupyterlab/mainmenu-extension@3.6.7
 - @jupyterlab/mainmenu@3.6.7
 - @jupyterlab/markdownviewer-extension@3.6.7
 - @jupyterlab/markdownviewer@3.6.7
 - @jupyterlab/mathjax2-extension@3.6.7
 - @jupyterlab/mathjax2@3.6.7
 - @jupyterlab/metapackage@3.6.7
 - @jupyterlab/nbconvert-css@3.6.7
 - @jupyterlab/nbformat@3.6.7
 - @jupyterlab/notebook-extension@3.6.7
 - @jupyterlab/notebook@3.6.7
 - @jupyterlab/observables@4.6.7
 - @jupyterlab/outputarea@3.6.7
 - @jupyterlab/pdf-extension@3.6.7
 - @jupyterlab/property-inspector@3.6.7
 - @jupyterlab/rendermime-extension@3.6.7
 - @jupyterlab/rendermime-interfaces@3.6.7
 - @jupyterlab/rendermime@3.6.7
 - @jupyterlab/running-extension@3.6.7
 - @jupyterlab/running@3.6.7
 - @jupyterlab/services@6.6.7
 - @jupyterlab/settingeditor-extension@3.6.7
 - @jupyterlab/settingeditor@3.6.7
 - @jupyterlab/settingregistry@3.6.7
 - @jupyterlab/shared-models@3.6.7
 - @jupyterlab/shortcuts-extension@3.6.7
 - @jupyterlab/statedb@3.6.7
 - @jupyterlab/statusbar-extension@3.6.7
 - @jupyterlab/statusbar@3.6.7
 - @jupyterlab/terminal-extension@3.6.7
 - @jupyterlab/terminal@3.6.7
 - @jupyterlab/theme-dark-extension@3.6.7
 - @jupyterlab/theme-light-extension@3.6.7
 - @jupyterlab/toc-extension@5.6.7
 - @jupyterlab/toc@5.6.7
 - @jupyterlab/tooltip-extension@3.6.7
 - @jupyterlab/tooltip@3.6.7
 - @jupyterlab/translation-extension@3.6.7
 - @jupyterlab/translation@3.6.7
 - @jupyterlab/ui-components-extension@3.6.7
 - @jupyterlab/ui-components@3.6.7
 - @jupyterlab/vdom-extension@3.6.7
 - @jupyterlab/vdom@3.6.7
 - @jupyterlab/vega5-extension@3.6.7
 - @jupyterlab/example-services-browser@3.6.7
 - node-example@3.6.7
 - @jupyterlab/example-services-outputarea@3.6.7
 - @jupyterlab/testutils@3.6.7

sigmarkarl committed:
[ci skip] bump version

sigmarkarl committed:
use promise for timeout

sigmarkarl committed:
jupter server from git

sigmarkarl committed:
bump version

sigmarkarl committed:
bump version

sigmarkarl committed:
original version

sigmarkarl committed:
original version

sigmarkarl committed:
original version
